runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/ClueScrollOverlay.java

```diff
@@ -85,6 +96,26 @@ public Dimension render(Graphics2D graphics)
 			}
 		}
 
+		if (clue.isRequiresLight() && ((plugin.getInventoryItems() != null && !HAS_LIGHT.fulfilledBy(plugin.getInventoryItems()) || (plugin.getEquippedItems() != null && !HAS_LIGHT.fulfilledBy(plugin.getEquippedItems())))))
+		{
+			if (clue instanceof EmoteClue)
+			{
+				if (((EmoteClue) clue).getLocationName().equals("Lumbridge swamp caves"))
+				{
+					for (Varbits varbit: BRAZIER_VARBITS)
```

Comment:
TIL that Mos Le'Harmless and Giant Mole are both entirely inside the Lumbridge Swamp Caves

For real though, you'll want another variable like the Spade and Light stuff, but this one will tell the overlay to look at a specific Varbit.

Reply:
I'm still trying to figure out the code base. I guess the question is do we want a variable for this clue scroll specifically (swamp caves)? There are a couple other clue scrolls I found that are coordinate based instead of emote based. Is that something to check for and is it done in a similar way?